src/main/java/nextstep/courses/domain/SessionStudent.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.courses.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import nextstep.users.domain.NsUser;
+
+public class SessionStudent {
+  List<NsUser> students;
+  private Integer maxStudent;
+
+  public SessionStudent() {
+    this(Integer.MAX_VALUE);
+  }
+  public SessionStudent(Integer maxStudent) {
+    this.maxStudent = maxStudent;
+    this.students = new ArrayList<>();
+  }
+
+  public  boolean canAcceptNewStudent() {
+    return students.size() < maxStudent;
```

Comment:
```suggestion
    return size() < maxStudent;
```

`List`의 `students` 사이즈를 물어보는것을 하나로 모아볼수도 있겠네요 😄